ch4/sec1/4-1-1.tex

```diff
@@ -12,11 +12,77 @@
 
 \setexercise{4}{1}{1}
 \begin{exercise}
-  
+Show that functors $F : C \to D$ and $G : D \to C$ and bijections $D(Fc, d) \cong C(c, Gd)$ for each $c \in C$ and $d \in D$ define an adjunction if and only if these bijections induce a bijection between commutative squares (4.1.4). That is, prove Lemma 4.1.3.
```

Comment:
We're trying to match the book's style, so sans serif for categories, for example.  There are macros \sC and \sD for the categories here. Also, `:` gives a relation; we should use `\colon` here; again, there's a macro for this: `$\func{F}{\sC}{\sD}$` does the right thing.
A nicety would be to include the statement of Lemma 4.1.3.